apps/microsoft-teams/frontend/src/hooks/useGetContentTypes.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useCallback, useState } from 'react';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { ConfigAppSDK } from '@contentful/app-sdk';
+import { ContentTypeProps } from 'contentful-management';
+
+/**
+ * This hook is used to get all the content types for the space
+ *
+ * @returns allContentTypes
+ */
+const useGetContentTypes = () => {
+  const [allContentTypes, setAllContentTypes] = useState<ContentTypeProps[]>([]);
+  const sdk = useSDK<ConfigAppSDK>();
+
+  const getAllContentTypes = useCallback(async () => {
+    try {
+      const contentTypesResponse = await sdk.cma.contentType.getMany({});
```

Comment:
How does pagination work here? I know we have a separate story to implement it but maybe stick a TODO that we're intentionally skipping pagination at this stage.